cfme/tests/cloud/test_providers.py

```diff
@@ -516,10 +518,10 @@ def test_cloud_names_grid_floating_ips(appliance, ec2_provider, soft_assert):
 @pytest.mark.tier(3)
 def test_display_network_topology(appliance, openstack_provider):
     """
-        Bugzilla:
-            1343553
+    Bugzilla:
+        1343553
 
-        Steps to Reproduce:
+    Steps to Reproduce:
```

Comment:
Not in scope of PR but I would like to request you just move in polarion docstring as `testSteps`